### ISSUE TITLE
When argument repeated via explicit indexes in `fmt.Errorf`, fails error wrap check

### ISSUE THREAD

**Issue by keiqu** [polyfloyd/go-errorlint#21]
Examples:
```go
func main() {
    fmt.Errorf("%[1]v %[1]v: %w", "value", errors.New("abc"))
}
```
```go
func main() {
    fmt.Errorf("%[1]v: %[1]w", errors.New("abc"))
}
```

Linting fails with the error: `non-wrapping format verb for fmt.Errorf. Use '%w' to format errors`.

**Comment by polyfloyd** [comment]
Yup, that should be supported. Will investigate when I have time
